src/devices/SenseHat/samples/SenseHat.Sample.cs

```diff
@@ -40,12 +45,29 @@ public static void Main(string[] args)
                     sh.Fill(n % 2 == 0 ? Color.DarkBlue : Color.DarkRed);
                     sh.SetPixel(x, y, Color.Yellow);
 
-                    Console.WriteLine($"Temperature: Sensor1: {sh.Temperature.Celsius} °C   Sensor2: {sh.Temperature2.Celsius} °C");
-                    Console.WriteLine($"Humidity: {sh.Humidity} %rH");
-                    Console.WriteLine($"Pressure: {sh.Pressure} hPa");
-                    Console.WriteLine($"Acceleration: {sh.Acceleration} g");
-                    Console.WriteLine($"Angular rate: {sh.AngularRate} DPS");
-                    Console.WriteLine($"Magnetic induction: {sh.MagneticInduction} gauss");
+                    var tempValue = sh.Temperature;
+                    var temp2Value = sh.Temperature2;
+                    var preValue = sh.Pressure;
+                    var humValue = sh.Humidity;
+                    var accValue = sh.Acceleration;
+                    var angValue = sh.AngularRate;
+                    var magValue = sh.MagneticInduction;
+                    var altValue = WeatherHelper.CalculateAltitude(preValue, defaultSeaLevelPressure, tempValue);
+
+                    Console.WriteLine($"Temperature Sensor 1: {tempValue.Celsius:0.#}\u00B0C");
+                    Console.WriteLine($"Temperature Sensor 2: {temp2Value.Celsius:0.#}\u00B0C");
+                    Console.WriteLine($"Pressure: {preValue:0.##}hPa");
+                    Console.WriteLine($"Altitude: {altValue:0.##}m");
+                    Console.WriteLine($"Acceleration: {sh.Acceleration}g");
+                    Console.WriteLine($"Angular rate: {sh.AngularRate}DPS");
+                    Console.WriteLine($"Magnetic induction: {sh.MagneticInduction}gauss");
+                    Console.WriteLine($"Relative humidity: {humValue:0.#}%");
+                    Console.WriteLine($"Heat index: {WeatherHelper.CalculateHeatIndex(tempValue, humValue).Celsius:0.#}\u00B0C");
+                    Console.WriteLine($"Summer simmer index: {WeatherHelper.CalculateSummerSimmerIndex(tempValue, humValue).Celsius:0.#}\u00B0C");
```

Comment:
I'd show just 1 parameter from weather helpers to showcase the usage but not more to not obscure the usage

Reply:
Again, maybe I should introduce a separator?

Reply:
Done (pending commit)